packages/mobile/src/store/product/selectors.ts

```diff
@@ -0,0 +1,7 @@
+import { RootState } from '~/common/types/types';
+
+const selectProduct = (state: RootState) => {
+  return state.product.product;
+};
```

Comment:
This belongs to products/current reducer

Reply:
left as it is for now